resources/schema.graphql

```diff
@@ -377,6 +383,10 @@ type DatasetFlowConfigsMut {
     paused: Boolean!
     batching: BatchingConditionInput!
   ): SetFlowConfigResult!
+  setConfigCompacting(
+    datasetFlowType: DatasetFlowType!
+    compactingArgs: CompactingConditionInput!
+  ): SetFlowConfigResult!
```

Comment:
Suggestion for you and Roma: let's split data types we return from different `setConfig...` methods, as different groups of errors are possible - as a result of current design, the fragments contain impossible variants.

Reply:
+